Tests/BootCamp.Chapter.Tests/PlayerTests.cs

```diff
@@ -0,0 +1,53 @@
+﻿using BootCamp.Chapter.Gambling;
+using System;
+using Xunit;
+
+namespace BootCamp.Chapter.Tests
+{
+    public class PlayerTests
+    {
+        [Fact]
+        public void New_Given_Arguments_For_Name_Sets_Name()
+        {
+            var name = "John Doe";
+            var hand = new Hand();
+
+            var player = new Player(name, hand);
+
+            Assert.Equal(player.Name, name);
+        }
+
+        [Fact]
+        public void New_Given_Arguments_For_Hand_Sets_Hand()
```

Comment:
Why do you need both tests? Both tests do the same thing, a lot of copy-pasting 😞 .
Just assert for both things in one test.

Reply:
make on test of it